### DISCUSSION TITLE
Do I need R.ID drivers for an RX 580 if ReBAR is working?

### DISCUSSION THREAD

**Discussion by MrAnonymous7122** [xCuri0/ReBarUEFI#193] [Q&A] [answered]
The official adrenalin drivers say that rebar cannot be enabled while gpuz shows a full size bar. Do I need R.ID drivers or is it fine with the normal amd drivers/

**Comment by xCuri0** [comment, marked as answer]
There's a registry edit you need to do (wiki says it) for it to work on official drivers 

R.ID drivers just let you enable it through a menu

**Comment by MrAnonymous7122** [comment]
Ok will do. How can I re enable rebar after resetting CMOS? I had to reset CMOS for other reasons but the time has reset and rebar has disabled.

**Reply by MrAnonymous7122** [reply]
Can I just set the time to the current date and it will fix itself?

**Reply by xCuri0** [reply]
yes
